user_manual/files/files.rst

```diff
@@ -352,6 +352,11 @@ Known Problems
   a value of **4294967295**, and reboot Windows or restart the **WebClient**
   service.
 
+**Problem**
+  Accessing your files from Microsoft Office via WebDAV fails.
+
+**Solution**
+  Known problems and their solutions are documented in the  `KB2123563<https://support.microsoft.com/kb/2123563/>`_ article.
```

Comment:
This link doesn't work. Just remove everything between `<` and `>`(including these) and create a link below like this:

```
.. _KB2123563: https://support.microsoft.com/kb/2123563/
```